### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | yes maybe
| Deprecations?   | no
| Related tickets | n/a
| Documentation   | n/a
| License         | MIT

#### What's in this PR?

Fixed some typos

### CODE REVIEW

File: Collector/DebugPluginCollector.php [hunk 1]
```diff
@@ -126,7 +126,7 @@ public function getFailedRequests()
      */
     public function getTotalRequests()
     {
-        return $this->getSucessfulRequests() + $this->getFailedRequests();
```
Comment (Nyholm) [inline Collector/DebugPluginCollector.php]: I did not wear my glasses when I was making this class.. Obviously...

**Comment by Nyholm** [comment]
Thank you for this PR. 

Can you make StyleCI happy as well?

**Comment by pborreli** [comment]
@Nyholm done.

**Comment by Nyholm** [comment]
Thank you!

**Pull request merged**